scripts/download-mapbox-gl-native-ios.sh

```diff
@@ -1,9 +1,16 @@
 #!/bin/sh
 
+cd ios/
+
 VERSION=$1
+CURRENT_VERSION=$(cat .framework_version)
```

Comment:
I like the flexibility this gives with being able to update the version independently. I do think we should provide a default version number for a few reasons

- Know the version of the Mapbox.framework that this lib was tested to work with.
- Allow users with not much experience in the mobile space to just be able to get there projects up and running without needing to add this file.

Reply:
For sure, I left that part in package.json the way it's currently done. This change was purely to prevent Mapbox.framework from being deleted/redownloaded each time there is a dependency change if the Mapbox verison wasn't changed. I added it because I was traveling this weekend and used a hotspot a lot, so deleting/redownloading ~60MB each time I added a new dependency got annoying. :-)

Reply:
ha, I misread it cool thanks for the PR!